projects/igniteui-angular/src/lib/tabs/tabs.component.ts

```diff
@@ -155,11 +164,32 @@ export class IgxTabsComponent implements IgxTabsBase, AfterViewInit, OnDestroy {
     /**
      * Provides an observable collection of all `IgxTabItemComponent`s.
      * ```typescript
-     * const tabItems = this.myTabComponent.tabs;
+     * const tabItems = this.myTabComponent.viewTabs;
      * ```
      */
     @ViewChildren(forwardRef(() => IgxTabItemComponent))
-    public tabs: QueryList<IgxTabItemComponent>;
+    public viewTabs: QueryList<IgxTabItemComponent>;
+
+    /**
+     * Provides an observable collection of all `IgxTabItemComponent`s.
+     * First try to get them as content children if not available get them as view children.
+     * ```typescript
+     * const tabItems = this.myTabComponent.tabs;
+     * ```
+     */
+    public get tabs(): QueryList<IgxTabItemComponent> {
+        if (this.hasContentTabs) {
+            return this.contentTabs;
+        }
+        return this.viewTabs;
+    }
+
+    /**
+     *@hidden
+     */
+    public get hasContentTabs(): boolean {
+        return (this.contentTabs && this.contentTabs.toArray().length > 0);
```

Comment:
contentTabs should have 'length' property, no need to cast to array.

Reply:
Addressed.